modules/ROOT/pages/whats-new-in-mule.adoc

```diff
@@ -124,6 +124,8 @@ The `greeting` function provides:
 * a work- specific message (`"Have a nice day at work!"`) on Tuesdays through Thursdays.
 * customized messages for the start and end of the week (Mondays and Fridays).
 
+Note that DataWeave uses the first function it finds that accepts the arguments of the function call, in the order that they are declared in the script.
```

Comment:
Do you want to make this an admonition? Or just remove the "Note that"? 

For both instances, I'm trying to parse the singular in the first half and the plural (they) in the second. Does the they refer to the arguments? Maybe clarify?

Reply:
ohhh, right. Overlooked "they". Good 👁️